File: ticketProject4-DistributedJava/TicketSystem/TicketNetworking/src/main/java/network/objectprotocol/GetAllARResponse.java
package network.objectprotocol;

import network.dto.ArtisticRepresentationDTO;

public class GetAllARResponse implements Response {
    private Iterable<ArtisticRepresentationDTO> arsDTO;

    public GetAllARResponse(Iterable<ArtisticRepresentationDTO> arsDTO) {
        this.arsDTO = arsDTO;
    }

    public Iterable<ArtisticRepresentationDTO> getARs() {
        return arsDTO;
    }
}

File: ticketProject7-CrossPlatform/TS-crossp/networking/response/GetAllARResponse.cs
using System;
using System.Collections.Generic;

namespace networking.protocol
{

	using ArtisticRepresentationDTO = networking.dto.ArtisticRepresentationDTO;

	[Serializable]
	public class GetAllARResponse : Response
	{
		private IEnumerable<ArtisticRepresentationDTO> arsDTO;

		public GetAllARResponse(IEnumerable<ArtisticRepresentationDTO> arsDTO)
		{
			this.arsDTO = arsDTO;
		}

		public virtual IEnumerable<ArtisticRepresentationDTO> getArs
		{
			get
			{
				return arsDTO;
			}
		}
	}


}